doc/outputs.md

```diff
@@ -73,7 +73,7 @@ $ openwec subscriptions edit <subscription> outputs add --format <format> files
 
 The Kafka driver sends events in a Kafka topic.
 
-For a given subscription, all events will be sent in the configured Kafka topic. You may want to add additionnal options to the inner Kafka client, such as `bootstrap.servers`. This options will be directly given to `librdkafka` (available options are listed here: https://docs.confluent.io/platform/current/clients/librdkafka/html/md_CONFIGURATION.html).
+For a given subscription, all events will be sent in the configured Kafka topic. If there are multiple topics then topic can contain that list, delimited with commas. You may want to add additionnal options to the inner Kafka client, such as `bootstrap.servers`. This options will be directly given to `librdkafka` (available options are listed here: https://docs.confluent.io/platform/current/clients/librdkafka/html/md_CONFIGURATION.html).
```

Comment:
The documentation should state that the events are distributed evenly among all the topics when a list of topics is provided.